dstep/Configuration.d

```diff
@@ -18,42 +18,71 @@ struct Configuration
     /// array of file names to translate to D
     string[] inputFiles;
 
-    /// expected programming language of input files
+    @("language|x", "Treat subsequent input files as having type <language>.")
     Language language;
 
+    @("version", "Show dstep version.")
+    bool dstepVersion;
+
+    @("clang-version", "Show libclang version.")
+    bool clangVersion;
+
     /// array of parameters needed to be forwarded to clang driver
     string[] clangParams;
 
     /// output file name or folder (in case there are many input files)
     string output;
 
-    /// strip all comments while translating
-    bool noComments;
-
-    /// package name
+    @("package", "Use <package> as package name.")
     string packageName;
 
-    /// use public imports for submodules
-    bool publicSubmodules;
+    @("comments", "Translate comments [default: true].")
```

Comment:
Hmm, actually I can do something like this: if user writes string like `[default]` it will be filled with the default value. The advantage over writing full default string manually, like `[default: true]` is the value will be kept in sync with real implementation.